package.json

```diff
@@ -445,7 +445,7 @@
   },
   "devDependencies": {
     "@commitlint/cli": "^12.1.4",
-    "@commitlint/config-conventional": "^12.1.4",
+    "@commitlint/config-angular": "^12.1.4",
```

Comment:
Why are we using an old version?

Reply:
v13 [has dropped Node 10 support](https://github.com/conventional-changelog/commitlint/compare/v12.1.4...v13.0.0), while Karma still supports Node 10. I guess it's less important for commitlint given that it only runs on Node 14 on CI, but thought it would be better to be consistent and upgrade all dependencies once we drop Node 10 support.

Reply:
OMG you know everything.